src/EventStore.Core/Index/TableIndex.cs

```diff
@@ -222,25 +229,77 @@ public void AddEntries(long commitPos, IList<IndexKey> entries)
                 {
                     prepareCheckpoint = Math.Max(prepareCheckpoint, collection[i].Position);
                 }
+                
+                TryProcessAwaitingTables(commitPos, prepareCheckpoint);
+            }
+        }
+     
+        public Task MergeIndexes()
+        {
+            TryManualMerge();
+            return Task.CompletedTask;
+
+            //return Task.Run(() =>
```

Comment:
This can probably be nuked?

Reply:
There is an outstanding item over surfacing this in the UI, but that doesn't need to happen to merge this. I suspect it will get nuked when I surface it in the admin page